packages/protocol/contracts/common/ExtractFunctionSignature.sol

```diff
@@ -0,0 +1,13 @@
+pragma solidity ^0.5.3;
+
+library ExtractFunctionSignature {
```

Comment:
I'm wondering about the usefulness of this being a library. Do we use this in any other contracts?

If we do keep it a library, let's make the function `internal`, not `public`, and not link it during migrations.

Reply:
Mostly it's in a library to make testing it easier...